playbooks/infrastructure/clevis.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Apply role clevis
+  hosts: "{{ hosts_clevis|default('clevis') }}"
+  serial: "{{ osism_serial['clevis']|default(osism_serial_default)|default(0) }}"
+
+  collections:
+    - osism.services
```

Comment:
This should be osism.commons and not osism.services.

Reply:
done